### ISSUE TITLE
Map H100 GPU on Azure

### ISSUE THREAD

**Issue by thomasjrademaker** [dstackai/gpuhunt#101]
There is no H100 mapping for Azure, however, a first H100 instance has appeared in the pricing list. Can you update the mapping please? 

<img width="1902" alt="Azure_H100_in_prices_list" src="https://github.com/user-attachments/assets/556effbb-81fe-4432-b0c3-9b4e7f80ec02">

Noting that DataCrunch will add H200 instances to its list too according to the website. I haven't checked in the API call, but I'd imagine that that should flow through soon as well.

**Comment by thomasjrademaker** [comment]
Hey guys, easy fix, just requires adding line 35 to [https://github.com/dstackai/gpuhunt/blob/main/src/gpuhunt/providers/azure.py](url)
`VMSeries(r"NC(\d+)adis_H100_v5", "H100", 80.0),  # NC H100 v5-series [H100 80GB]`

**Comment by peterschmidt85** [comment]
cc @r4victor

**Comment by r4victor** [comment]
@thomasjrademaker, we do not have access to h100 on azure to test it internally. Do you have the access?

**Comment by thomasjrademaker** [comment]
@jvstme yep, just gpuhunt, no dstack. Great, just including in gpuhunt works for me!
